index.html

```diff
@@ -206,28 +225,34 @@
 				// },
 				//Phường 4
 				//Example
-				// {
-				// 	type: "polygon",
-				// 	rings: [
-				// 		[106.80514114445627, 10.870158436990936],
-				// 		[106.80400388783394, 10.868915147546755],
-				// 		[106.8032099539655, 10.868325110065758],
-				// 		[106.80245893544132, 10.869210165849735],
-				// 		[106.80245893544132, 10.869484111155634],
-				// 		[106.8037463957685, 10.871085632526057]
-				// 	],
-				// 	Name: "Đại học CNTT",
-				// 	Location: "Khu phố 6, P.Linh Trung, Q.Thủ Đức, Tp.Hồ Chí Minh",
-				// 	symbol: {
-				// 		type: "simple-fill",
-				// 		color: [227, 139, 79, 0.4],
-				// 		outline: {
-				// 			color: [255, 255, 255],
-				// 			width: 1
-				// 		}
-				// 	},
-				// 	popupTemplate: point_template_university
-				// },
+				{
+					type: "polygon",
+					rings: [
+						[106.68209335660033, 10.770074710273892],
+						[106.67904430070908, 10.772709893579371], 
+						[106.68282879153882, 10.775936355835066], 
+						[106.68271611794627, 10.775984486566879], 
+						[106.68265659836406, 10.776052516972653], 
+						[106.68175874717366, 10.777547601177377], 
+						[106.68184727867835, 10.777598757636863], 
+						[106.68192087832313, 10.777675073803865], 
+						[106.68638256253398, 10.775275491824537], 
+						[106.68682577057525, 10.775030447952853], 
+						[106.68481506507447, 10.772919119592833], 
+						[106.68209335660033, 10.770074710273892]
+					],

```

Comment:
I think this list of coordinates is not really complete. Please zoom in on Google Maps to the maximum and get all the coordinates of the zigzag lines within your ability.

Reply:
i had zoom and adjusted Coordinates